src/JMEJetAnalyzer.cc

```diff
@@ -194,6 +200,17 @@ void JMEJetAnalyzer::analyze(const edm::Event& iEvent,
 
      jtarea.push_back(jet.jetArea());
      jtjec.push_back(jet.jecFactor(0));
+     jtpt.push_back( jet.pt() );
+     jteta.push_back( jet.eta() );
+     
```

Comment:
These two variables are also already in the tree. See the ``p4`` variable. If it's easier for whatever reason to have 4 branches (pt eta phi and E) instead of one LorentzVector, then we should modify the base class instead of duplicate the information.